src/Libraries/RevitNodes/GeometryConversion/UnitConverter.cs

```diff
@@ -12,11 +12,13 @@ public static class UnitConverter
     {
         public static double DynamoToHostFactor(ForgeTypeId specTypeId)
         {
-            var unitTypeId =
-                DocumentManager.Instance.CurrentDBDocument.GetUnits()
-                    .GetFormatOptions(specTypeId)
-                    .GetUnitTypeId();
-           
+            var units = DocumentManager.Instance.CurrentDBDocument.GetUnits();
+            if (!UnitUtils.IsMeasurableSpec(specTypeId))
+            {
+                return 1;
+            }
+
```

Comment:
can we just add the condition check first in this function and no change to later codes.